driver/ppm_flag_helpers.h

```diff
@@ -1822,6 +1825,31 @@ static __always_inline u32 chmod_mode_to_scap(unsigned long modes)
 	return res;
 }
 
+static __always_inline u32 umount2_flags_to_scap(unsigned long flags)
+{
+	u32 res = 0;
+
+#ifdef __NR_umount2
+#ifdef MNT_FORCE
```

Comment:
Please note that  `MNT_FORCE` and other flags are not defined in the modern probe since we cannot include `include <linux/fs.h>` so you need to manually add them here https://github.com/falcosecurity/libs/blob/0347a329fe18c2724d4c6a8ce9a6224105d670e5/driver/modern_bpf/definitions/missing_definitions.h#L554 otherwise tests won't pass